cogboard-webapp/src/components/widgets/types/TextWidget/index.js

```diff
@@ -1,16 +1,53 @@
-import React from 'react';
+import React, { useRef } from 'react';
+import { useSize } from 'react-hook-size';
 import { bool, string } from 'prop-types';
 
-import { Typography } from '@material-ui/core';
-import { StyledPre, VerticalText } from './styled';
+import {
+  TypographyVariant,
+  CenterWrapper,
+  StyledPre,
+  RotatedStyledPre,
+  OverflowingText,
+  SetWidth
+} from './styled';
+
+export const ModifiedWidth = (component, height) => {

```

Comment:
Imho helper functions/hooks names should start with lowercase

Reply:
Sorry, now I realised that these are subcomponents - maybe their names are a bit misleading

Reply:
What names would fit them better in your opinion?

Reply:
Maybe turn it into some kind of HOC and add `width` prefix... I don't have a clear idea for these names - for me they are totally ok